nservicebus/upgrades/5to6.md

```diff
@@ -37,6 +37,11 @@ snippet: 5to6-endpointNameRequired
 The endpoint name is used as a logical identifier when sending or receiving messages. It is also used for determining the name of the input queue the endpoint will be bound to. See [Derived endpoint name](/nservicebus/upgrades/5to6-endpoint-name-helper.md) for the algorithm used in Versions 5 and below to select endpoint name if backwards compatibility is a concern.
 
 
+## OverrideLocalAddress is removed
+
+`OverrideLocalAddress` was replaced with `OverrideInputQueueName`. The specified parameter will be used as the name of the local input queue.
```

Comment:
can u show some snippets here to give some more context. and link to the doco for OverrideLocalAddress